### PR TITLE
feat: table merged cell plugin(markdown)

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
* added merged table cell plugin
  * markdown parser, renderer
  * apply jest, typescript, snowpack env

스크린샷 2021-04-05 오후 4 48 41

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: plugins/table-merged-cell/src/markdown/parser.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  return [spanCount, content];
+}
+
+function extendTableCellIndexWithRowspanMap(
+  node: MergedTableCell,
+  parent: MergedTableRow,
+  rowspan: number
+) {
+  const prevRow = parent.prev;
+  const columnLen = parent.parent.parent.columns.length;
```
Comment (dotaitch) [inline plugins/table-merged-cell/src/markdown/parser.ts:23]: `columnLen`은 아래 `if (prevRow) { ... }` 블럭에서만 사용되는데, if 블럭 안에서 선언하는 건 어떤가요?

**Review by dotaitch — APPROVED** [review]
리뷰 완료했습니다.

File: plugins/table-merged-cell/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { markdownParsers } from './markdown/parser';
+import { toHTMLRenderers } from './markdown/renderer';
```
Comment (lja1018) [inline plugins/table-merged-cell/src/index.ts:2]: `@/markdown/~~`은 어떤가요?

File: plugins/table-merged-cell/src/markdown/parser.ts [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        if (i <= node.endIdx) {
+          node.startIdx += 1;
+          node.endIdx += 1;
+        }
+      }
+    }
+  }
+
+  if (rowspan > 1) {
+    for (let i = node.startIdx; i <= node.endIdx; i += 1) {
```
Comment (lja1018) [inline plugins/table-merged-cell/src/markdown/parser.ts:42]: `node.endIdx` 변수로 사용하는건 어떤가요?

File: plugins/table-merged-cell/src/markdown/parser.ts [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        node.colspan = colspan;
+        node.endIdx += colspan - 1;
+      }
+      if (rowspan > 1) {
+        node.rowspan = rowspan;
+      }
+
+      extendTableCellIndexWithRowspanMap(node, parent, rowspan);
+
+      const tablePart = parent!.parent!;
```
Comment (lja1018) [inline plugins/table-merged-cell/src/markdown/parser.ts:92]: `parent.parent`로 해도 될것 같습니다!

File: plugins/table-merged-cell/tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  "compilerOptions": {
+    "baseUrl": ".",
+    "importHelpers": false,
+    "paths": {
+      "@/*": ["src/*"],
+      "@t/*": ["types/*"]
+    },
+    "lib": ["esnext", "dom", "dom.iterable"]
+  }
+}
```
Comment (lja1018) [inline plugins/table-merged-cell/tsconfig.json:14]: 엔터가 필요할것 같습니다 ㅋㅋ

**Review by lja1018 — APPROVED** [review]
리뷰완료입니다.

File: plugins/table-merged-cell/__test__/integration/markdownMergedTable.spec.ts [hunk 6]
```diff
@@ -0,0 +1,580 @@
...
+  }
+
+  beforeEach(() => {
+    container = document.createElement('div');
+    document.body.appendChild(container);
+
+    editor = new Editor({
+      el: container,
+      previewStyle: 'vertical',
+      // @ts-ignore
```
Comment (jungeun-cho) [inline plugins/table-merged-cell/__test__/integration/markdownMergedTable.spec.ts:28]: 에러가 나나요?

**Review by jungeun-cho — APPROVED** [review]
수고하셨습니다. 리뷰 완료했습니다.

**Pull request merged**